apps/sms/test/marionette/conversation_test.js

```diff
@@ -154,11 +154,11 @@ marionette('Conversation Panel Tests', function() {
           type: 'Mobile'
         }]
       }]);
+
+      messagesApp.Inbox.findConversation(threads[1].id).tap();
```

Comment:
I want to remove ```firstConversation``` for a long time already (and remove it in my drafts dialog patch), for the following reason: sometimes threads can be generated with the same timestamp (using Date.now) so that the order in which they appear  can change from run to run that leads to intermittents. So here I've just used the chance to do that.